all-time/all-time.js

```diff
@@ -0,0 +1,254 @@
+import { getFromLocalStorage, findById } from '../functions.js';
+import { pokedex } from '../pokemon.js';
+
+const playAgainButton = document.getElementById('play-again');
+const results = document.getElementById('results');
+const allTimeSessionsDataLS = 'allTimeSessionsDataLS';
+
+
+//Need click handler to go back to main page and play again
+playAgainButton.addEventListener('click', () => {
+    window.location.href = '../';
+});
+
+//Chart data build
+const allTimePokemon = [];
+const allTimeEncounters = [];
+const allTimeCaptures = [];
+const allTimeLabels = [];
+const allTimeSessionsData = getFromLocalStorage(allTimeSessionsDataLS);
+
+
+//Build pokemon array
+for (let i = 0; i < pokedex.length; i++) {
+    allTimePokemon.push({ id: pokedex[i].id,
+        name: pokedex[i].pokemon,
+        encounters: 0,
+        captures: 0,
+    });      
+}
+
+
+//Add all encounters and captures to pokemon array
+for (let i = 0; i < allTimeSessionsData.length; i++) {
```

Comment:
nice! Definitely a unique way to solve this with a nested loop again. Great work!